nni/compression/pytorch/speedup/quantization_speedup/frontend_to_onnx.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import torch
+import torch.nn.functional as F
+from torchvision import datasets, transforms
+from torch.autograd import Variable
+import onnx
+import onnx.numpy_helper
+
+class LayernameModuleWrapper(torch.nn.Module):
```

Comment:
add docstring to explain this class and why we need this class

Reply:
Added.